main_test.go

```diff
@@ -66,7 +73,7 @@ func (f *fakeSecrets) Create(secret *v1.Secret) (*v1.Secret, error) {
 	_, ok := f.store[secret.Name]
 
 	if ok {
-		return nil, fmt.Errorf("Secret %v already exists", secret.Name)
+		return nil, fmt.Errorf("secret %v already exists", secret.Name)
```

Comment:
I assume all these case changes from `Secret` to `secret` were a global find/replace?

Reply:
They weren't, unfortunately. I can run thru the source once more to update any that were missed if needed. :)

Reply:
Nah no big deal, there are too many other inconsistencies to pick on logging. =)

Thanks for the PR!!!!

Reply:
:D Glad to help!